AndroidSDKTests/src/test/java/com/leanplum/internal/RequestTest.java

```diff
@@ -67,6 +68,20 @@ public void setUp() {
     Leanplum.setApplicationContext(context);
   }
 
+  /**
+   * Test that read writes happened sequentially when calling sendNow().
+   */
+  @Test
+  public void testSequenceOfDatabaseOperations() {
+    Map<String, Object> params = new HashMap<>();
+    params.put("data1", "value1");
+    params.put("data2", "value2");
+    ThreadWaiter waiter = new ThreadWaiter();
+    Request request = new Request("POST", Constants.Methods.START, params, waiter);
```

Comment:
is there a an existing const for "POST" string ?

Reply:
I will add it